File: front/minhas-despesas/src/models/despesa-create-dto.ts
export interface DespesaCreateDto {
    despesaId?: string;
    periodoId: string;
    grupoDespesaId: string;
    dataCadastro?: Date;
    dataVencimento?: Date;
    dataQuitacao?: Date;
    descricao: string;
    valor: number;    
}

File: back/MinhasDespesas/MinhasDespesas.Models/Dtos/DespesaCreateDto.cs
namespace MinhasDespesas.Models.Dtos;

public class DespesaCreateDto
{
    public Guid? DespesaId { get; set; }
    public Guid PeriodoId { get; set; }
    public Guid GrupoDespesaId { get; set; }
    public DateTime? DataVencimento { get; set; }
    public DateTime? DataQuitacao { get; set; }
    public string Descricao { get; set; }
    public double Valor { get; set; }
}